cmd/registry/cmd/compute-lintstats.go

```diff
@@ -31,13 +31,22 @@ import (
 
 func init() {
 	computeCmd.AddCommand(computeLintStatsCmd)
+	computeLintStatsCmd.Flags().String("linter", "", "name of linter associated with these lintstats (aip, spectral, gnostic)")
 }
 
 var computeLintStatsCmd = &cobra.Command{
 	Use:   "lintstats",
 	Short: "Compute summaries of linter runs",
 	Args:  cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		linter, err := cmd.LocalFlags().GetString("linter")
+		if err != nil { // ignore errors
+			linter = ""
+		}
+		if linter == "" {
+			log.Fatalf("Please specify a linter with the --linter flag")
+		}
```

Comment:
If I'm reading this correctly, it's equivalent to `if err != nil || linter == "" { log.Fatalf(...) }`.

Reply:
Agreed, updated.